src/Components/Web/src/Properties/ILLink.Descriptors.xml

```diff
@@ -0,0 +1,22 @@
+<linker>
+  <assembly fullname="Microsoft.AspNetCore.Components.Web">
+    <type fullname="Microsoft.AspNetCore.Components.Forms.EditForm" required="false" preserve="all" />
```

Comment:
Do we really need the whole Type of all these classes?

Reply:
These are all components, and preserving these entirely is the safest course of action at this point.

Reply:
The ones for `M.A.Components` don't have `preserve="all"`. Is the difference just that it's more work (and maintenance cost) to reason about the exact preservability rules for all these extra components in `.Web`?

Reply:
I'll make the two consistent. `preserve="all"` is the default:

```
<!-- No "preserve" attribute and no members specified means preserve all members -->
<type fullname="Assembly.B" /> 
```

https://github.com/mono/linker/blob/main/docs/data-formats.md#preserve-a-type